R/data.R

```diff
@@ -27,26 +27,31 @@
 ##' @param trim_pillar2 The number of days of pillar 2 data to trim to avoid
 ##'   back-fill issues. We typically use a value of 7 days.
 ##'
+##' @param admissions_backfill A date string to calculate the number of days
+##'   of admissions by age to trim to avoid back-fill issues. Aggregate
+##'   data will instead be used for that same number of days.
```

Comment:
This could be a lot clearer - is it the last date we use admissions by age, or the first date we don't? Also we usually have "date" in the names of date parameters, so we probably should do so here for consistency.